core/templates/components/question-directives/modal-templates/question-editor-modal.directive.html

```diff
@@ -28,13 +28,16 @@ <h4 ng-if="!newQuestionIsBeingCreated" style="margin-top: 4vh;"> <strong> Questi
                    can-edit-question="canEditQuestion"
                    question-changed="questionChanged">
   </question-editor>
-  <div class="alert alert-danger" ng-if="validationError || (question.getStateData().interaction.id && isSaveButtonDisabled())">
-    <angular-html-bind html-data="validationError || isSaveButtonDisabled()"></angular-html-bind>
-    <span class="oppia-transition-200"
-          ng-click="removeErrors()"
-          style="float: right;">
-      <i ng-show="validationError" class="material-icons md-18">&#xE5CD;</i>
-    </span>
+  <div class="alert alert-danger" ng-if="question.getStateData().interaction.id && question.validate()">
```

Comment:
Perhaps question.getValidationErrorMessages() is a better name -- question.validate() sounds like a function that returns true/false.